examples/advanced_diffusion_training/train_dreambooth_lora_sd15_advanced.py

```diff
@@ -202,9 +183,27 @@ def save_model_card(
 Special VAE used for training: {vae_path}.
 
 """
-    with open(os.path.join(repo_folder, "README.md"), "w") as f:
-        f.write(yaml + model_card)
-
+    model_card = load_or_create_model_card(
+        repo_id_or_path=repo_id,
+        from_training=True,
+        license="openrail++",
+        base_model=base_model,
+        prompt=instance_prompt,
+        model_description=model_description,
+        inference=True,
+        widget=widget_dict,
+    )
+
+    tags = ["text-to-image", 
+            "diffusers", 
+            "diffusers-training",
+            "lora",
```

Comment:
since it can be either a LoRA or a DoRA, we need to check if the `use_dora` arg is True and adapt the tag accordingly 
(we check that in the line - `lora = "lora" if not use_dora else "dora"` :) )

Reply:
got it, my bad for removing the check! thanks for pointing out, pushing it now.

Reply:
lets just switch `"lora"` in the tags here to `lora` and we're good to go 😊